source/io/HttpClient.cpp

```diff
@@ -446,6 +447,21 @@ VIREO_FUNCTION_SIGNATURE10(HttpClientPost, UInt32, StringRef, StringRef, StringR
 }
 
 //------------------------------------------------------------
+// handle(0), supply credentials(1), error cluster(2)
+VIREO_FUNCTION_SIGNATURE3(HttpClientSetCredentials, UInt32, Boolean, ErrorCluster)
+{
+#if kVireoOS_emscripten
+   jsHttpClientSetCredentials(
```

Comment:
You may need to sync, @cglzaguilar added a new pattern for error handling that should be used here